miniapp/USERGUIDE.md

```diff
@@ -612,6 +612,80 @@ dependency {
 ```
 </details>
 
+<details><summary markdown="span"><b>How can I use this SDK in a Java project?</b>
+</summary>
+
+We don't support usage of the Mini App SDK in a Java project. This is because this SDK uses Kotlin specific features such as `suspend` functions and `Coroutines`.
+
+However, it is possible to use this SDK in a Java project if you wrap the calls to `suspend` functions into something that Java can use such as a `Future` or a normal callback interface.
+
+To do this, you will need to integrate Kotlin and Kotlin Coroutines into your project:
+
+```groovy
+// In your top level "build.gradle" file:
+buildscript {
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50"
+    }
+}
+
+// In your project "myApp/build.gradle" file
+apply plugin: 'kotlin-android'
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.50"
```

Comment:
Looks like if the hostapps do `implementation project` of miniapp SDK, then they can get this library because the same dependency has been declared in miniapp SDK. But, still we can just mention it as it's required to integrate Kotlin.

Reply:
We have this set as an `implementation` dependency in our SDK rather than an `api` dependency, so the Host App wouldn't get this dependency on their own compile classpath. 

Anyways, I think that it's a good practice for Apps to directly have dependencies declared which they use in their own code rather than relying on a transitive dependency. For example, the transitive dependency could be removed and then their own code would be broken (although that's really unlikely to happen in the case of the kotlin-stdlb dependency).